src/environments/environment.prestage.ts

```diff
@@ -0,0 +1,16 @@
+import { EnvironmentConfig, EnvironmentType } from './model';
+
+export const environment: EnvironmentConfig = {
+  production: true,
+  environmentType: EnvironmentType.PRESTAGE,
+  authConfig: {
+    issuer: `${window.location.origin}/auth/realms/OGLOSZENIA`,
+    showDebugInformation: false,
+    redirectUri: `${window.location.origin}/ogloszenia/`,
+    logoutUrl: `${window.location.origin}/ogloszenia/`,
+  },
+  applicationInsightsConnectionString: '%BACKEND_REPLACED_APPLICATION_INSIGHTS_CONNECTION_STRING%',
+  recaptcha: {
+    siteKey: '',
```

Comment:
Nie potrzebujemy klucza?

Reply:
Nie chciałem użyć tego samego co na prod, jeśli ma być ten sam to oczywiście mogę go dodać.

Reply:
Zrobiłem update zgodnie ze zgłoszeniem.